pkg/cloud/vsphere/provisioner/govmomi/utils.go

```diff
@@ -67,11 +65,57 @@ func (pv *Provisioner) GetKubeadmToken(cluster *clusterv1.Cluster) (string, erro
 	ncluster.ObjectMeta.Annotations[constants.KubeadmTokenExpiryTime] = time.Now().Add(constants.KubeadmTokenTtl).Format(time.RFC3339)
 	_, err = pv.clusterV1alpha1.Clusters(cluster.Namespace).Update(ncluster)
 	if err != nil {
-		glog.Infof("Could not cache the kubeadm token on cluster object: %s", err)
+		glog.Infof("Could not cache the kubeadm token on cluster object: %s", err.Error())
 	}
 	return token, err
 }
 
+// The logic for this function comes from various places in kubeadm itself
+func (pv *Provisioner) createKubeadmToken() (string, error) {
+	// Create the actual token string
+	token, err := tokenutil.GenerateBootstrapToken()
+	if err != nil || token == "" ||
+		len(token) < (tokenapi.BootstrapTokenIDBytes + tokenapi.BootstrapTokenSecretBytes) {
+		return "", fmt.Errorf("unable to create kubeadm token: %s", err.Error())
```

Comment:
cosmetic, but no need to call `Error()`, just `err` is fine (more cases of this below).